src/Client.ts

```diff
@@ -206,7 +210,10 @@ export function defaultOptions(opts?: Partial<ClientOptions>): ClientOptions {
     maxContentSize: MaxContentSize,
     persistConversations: true,
     skipContactPublishing: false,
-    basePersistence: new LocalStoragePersistence(),
+    basePersistence:
+      typeof localStorage === 'undefined'
```

Comment:
can we use something less brittle than a magic variable on `global`/`globalThis` to determine the environment?

this seems to be a common way of determining if we're in a browser:

```
const isBrowser =
  typeof window !== "undefined" && typeof window.document !== "undefined";
```

WDYT?

Reply:
I'm good with that